contrib/ros/envs/sawyer/push_env.py

```diff
@@ -2,51 +2,156 @@
 Push task for the sawyer robot
 """
 
+from geometry_msgs.msg import Pose, Point
 import numpy as np
 
 from rllab.core.serializable import Serializable
+from rllab.envs.base import Step
+from rllab.spaces import Box
 
-from contrib.ros.envs import sawyer_env
+from contrib.ros.envs.ros_env import RosEnv
 from contrib.ros.robots.sawyer import Sawyer
+from contrib.ros.util.common import rate_limited
 
-INITIAL_ROBOT_JOINT_POS = {
-    'right_j0': -0.041662954890248294,
-    'right_j1': -1.0258291091425074,
-    'right_j2': 0.0293680414401436,
-    'right_j3': 2.17518162913313,
-    'right_j4': -0.06703022873354225,
-    'right_j5': 0.3968371433926965,
-    'right_j6': 1.7659649178699421,
-}
 
-
-class PushEnv(sawyer_env.SawyerEnv, Serializable):
+class PushEnv(RosEnv, Serializable):
     def __init__(self,
                  initial_goal,
                  task_obj_mgr,
```

Comment:
I don't understand why I have to construct a `TaskObjectManager` to have a `PushEnv`. Perhaps I would want some fine-grained control over objects in some cases and I'd live an override, but the general design pattern for environments is that they are self-contained.

If I want a `PushEnv` and don't care a lot about custom objects, why are you forcing me to learn about `TaskObjectManager`?